ci/operations/scale_out_cf_for_app-autoscaler.yaml

```diff
@@ -13,3 +13,13 @@
   path: /instance_groups/name=diego-cell/vm_extensions
   value:
   - 500GB_ephemeral_disk
+
+
+- type: replace
+  path: /instance_groups/name=cc-worker/jobs/name=cloud_controller_worker/properties/cc/maximum_app_disk_in_mb?
+  value: 4096
```

Comment:
increase the disk limit each app instance can get in the CF.

WHY: because we deploy multiple cf apps as mtar